pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/prometheus/monitor.go

```diff
@@ -47,6 +48,10 @@ kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
```

Comment:
I do not think we can remove it from kustomize/v1 
However, could we remove these ones from kustomize/v2-alpha?
I think is valid we create an issue to check it the alpha plugin

Reply:
I wonder if this would be best in a follow-up (we could potentially do it in this pr but I think we should keep the scope to adding the recommended labels)? I think it is a good idea to remove it as well, but I am not sure which label would be best to replace it in the various selector fields. We should know more as to which label is common to all manifests after this PR is done (although i think it is likely that the label we will be able to use is `app.kubernetes.io/part-of` and map it to the project name).

WDYT?

Reply:
I do not think that we should do this one here @everettraven.
I think we can only track an issue and check it after.  
Also, to remove this one I think we can only to do in the kustomize/v2-alpha